applications/asset_tracker_v2/boards/nrf9160dk_nrf9160ns.overlay

```diff
@@ -0,0 +1,44 @@
+/	{
+	/delete-node/ leds;
+
+	leds0 {
+		compatible = "gpio-leds";
+		status = "okay";
+		label = "LED0";
+		led0: led_0 {
+			status = "okay";
+			gpios = <&gpio0 2 0>;
+			label = "Green LED 1";
```

Comment:
@anangl Can i ask you for opinion here? Shall we stay with `LED 1` or change it back to `LED 0`? In <board>_common.dts file its like here so i asked @zycz to leave it as is but im thinking if its good or not right now.
Another question for naming this node led**s**0 vs led0.

Reply:
Same goes for thingy91.overlay. Coudl you have a look @anangl?

Reply:
We cannot use generic values - CAF leds define one logical led as a gpio-leds node. Personally I don't care if this is called leds0 or led0 as long as it is consistent. 

As for using "Green LED 1" instead of Green LED 0" it's a matter of how humans look at things since label is a human readable form. I think it does not matter really.